src/utils/useTimeZoneUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { useOrganization, useTimeZones, useUser } from 'src/providers';
+
+import { TimeZoneDescription } from 'src/types/TimeZones';
+
+const DEFAULT_UTC = { id: 'Etc/UTC', longName: 'Coordinated Universal Time' };
+
+const getTimeZone = (timeZones: TimeZoneDescription[], id?: string): TimeZoneDescription | undefined => {
+  if (!id) {
+    return undefined;
+  }
+  return timeZones.find((tz) => tz.id === id);
+};
+
+/**
+ * Helper function to return UTC as per the supported time zones list
+ */
+export const getUTC = (timeZones: TimeZoneDescription[]): TimeZoneDescription => {
+  return getTimeZone(timeZones, DEFAULT_UTC.id) ?? DEFAULT_UTC;
+};
+
+/**
+ * Get a fallback time zone (based on org, user, etc.)
+ */
+export const useDefaultTimeZone = () => {
```

Comment:
this is needed in places where we initialize an entity to a time zone (if it doesn't already have one)